app/client/ui/CustomSectionConfig.ts

```diff
@@ -4,7 +4,7 @@ import * as kf from 'app/client/lib/koForm';
 import {ColumnToMapImpl} from 'app/client/models/ColumnToMap';
 import {ColumnRec, ViewSectionRec} from 'app/client/models/DocModel';
 import {reportError} from 'app/client/models/errors';
-import {cssLabel, cssRow, cssSeparator} from 'app/client/ui/RightPanelStyles';
+import {cssLabel, cssHelp, cssRow, cssSeparator} from 'app/client/ui/RightPanelStyles';
```

Comment:
Sorry for nitpicking, but can you move it to the beginning (so it is alphabetically sorted)?

Reply:
Done and rebased :)